src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -693,13 +688,14 @@ private async Task<ContentValidateResult> CreateContentItemVersionAsync(ContentI
             // Invoked create handlers.
             var context = new CreateContentContext(contentItem);
             await Handlers.InvokeAsync((handler, context) => handler.CreatingAsync(context), context, _logger);
-            await ReversedHandlers.InvokeAsync((handler, context) => handler.CreatedAsync(context), context, _logger);
 
             // The content item should be placed in the session store so that further calls
             // to ContentManager.Get by a scoped index provider will resolve the imported item correctly.
             _session.Save(contentItem);
```

Comment:
Before `CreatedAsync()` handlers as it has not been queried, so not yet in the identity map